.github/workflows/ci.yaml

```diff
@@ -5,7 +5,7 @@ on:
       - develop
       - release/**
       - hotfix/**
-      - feat/dev-583-remove-set-env-in-github-actions
+      - feat/dev-2469-fix-github-action
```

Comment:
Is this just for testing? Do we plan to remove this before merging?

Reply:
This is for testing, but this should not matter even if we merged it in production.